packages/autorest.python/tasks.py

```diff
@@ -536,6 +537,14 @@ def regenerate_custom_poller_pager_legacy(c, debug=False):
     )
     _run_autorest([cmd], debug=debug)
 
+@task
+def regenerate_mixed_api_version_legacy(c, debug=False):
```

Comment:
It is very easy to reproduce the error as long as we combined files which have different api-version into one tag. So I use two existing swagger files `head.json` and `azure-special-properties.json`  to test it